Doc/library/typing.rst

```diff
@@ -423,8 +430,8 @@ to this is that a list of types can be used to substitute a :class:`ParamSpec`::
    >>> Z[int, [dict, float]]
    __main__.Z[int, (<class 'dict'>, <class 'float'>)]
 
-
-Furthermore, a generic with only one parameter specification variable will accept
+Another difference between :class:`TypeVar` and :class:`ParamSpec` is that a
+generic with only one parameter specification variable will accept
```

Comment:
Not sure this needs to be changed in 3.11. The reason why we changed it in 3.12+ was because we added another paragraph immediately preceding this one, which broke the flow; starting the paragraph with "Furthermore" didn't work anymore. But in 3.11, we're not adding the new paragraph, so "Furthermore" works fine, and is more concise. I'd revert this change:

```suggestion
Furthermore, a generic with only one parameter specification variable will accept
```